test/integration/storage/StorageConfig.test.js

```diff
@@ -81,7 +90,7 @@ describe('StorageConfig', () => {
             id: publisherAccount.address + '/StorageConfigTest/' + Date.now()
         })
         await addStreamToStorageNode(stream.id, storageNodeAccount.address, client)
-        await storageNode.refreshStorageConfig()
+        await waitForCondition(() => isPersistent(stream.id))
```

Comment:
Might wanna tinker with `refreshInterval` of waitForCondition, otherwise awful many queries being done :D